### PR TITLE
35 restmqtt update gateway processes to respond to configuration change and to use mqtt clients

### PR DESCRIPTION
**Pull request by stekap000**
MQTT clients are added to processes (one for each process since that is the way that original code handled mqtt on sensor side).
Additional MQTT client in added in main for final stats_service data.
Function calls to data_service functions are modified to accept additional parameters (username and gateway-cloud broker client).

### CODE REVIEW

**Review by CodeBojan — APPROVED** [review]

**Review by CodeBojan — DISMISSED** [review]

**Review by CodeBojan — APPROVED** [review]

**Review by sergiPopescou — CHANGES_REQUESTED** [review]

File: src/app.py [hunk 1]
```diff
@@ -68,11 +68,8 @@
 qos: int
     Quality of service of MQTT.
 '''
-
 import json
```
[the remark this answers is not in the document]
Comment (stekap000) [reply src/app.py]: Ne koristi se, dio je originalnog koda. Uklonjeno je.

File: src/app.py [hunk 2]
```diff
@@ -81,7 +78,8 @@
 import paho.mqtt.client as mqtt
 from multiprocessing import Process, Queue, Event
 from threading import Thread
-
+from mqtt_util import *
```
[the remark this answers is not in the document]
Comment (stekap000) [reply src/app.py]: Promijenjeno.

File: src/app.py [hunk 3]
```diff
@@ -81,7 +78,8 @@
 import paho.mqtt.client as mqtt
 from multiprocessing import Process, Queue, Event
```
[the remark this answers is not in the document]
Comment (stekap000) [reply src/app.py:78]: Koristi se u main funkciji. Isto je dio originalnog koda.

File: src/app.py [hunk 4]
```diff
@@ -344,37 +343,60 @@ def on_message_handler(client, userdata, message):
...
-                        broker_address=config[mqtt_broker][address],
-                        broker_port=config[mqtt_broker][port],
-                        keepalive=config[temp_settings][interval] * 3,
-                        infoLogger=infoLogger,
-                        errorLogger=errorLogger,
-                        flag=flag,
-                        sensor_type="TEMP",
-                        bus=None,
-                        )
+    # [REST/MQTT]
```
Comment (sale3) [inline src/app.py:346]: Treba izbaciti ovaj zakomentarisani dio.

**Review by sale3 — CHANGES_REQUESTED** [review]

**Review by sergiPopescou — APPROVED** [review]

**Comment by sergiPopescou** [comment]
So proud of this community - meme

**Review by sale3 — APPROVED** [review]

**Pull request merged**